build/build-modules-js/init/common/copy-all-files.es6.js

```diff
@@ -15,5 +15,5 @@ const RootPath = process.cwd();
 module.exports.copyAllFiles = async (dirName, name, type) => {
   const folderName = dirName === '/' ? '/' : `/${dirName}`;
   await copy(join(RootPath, `node_modules/${name}/${folderName}`),
-    join(RootPath, `media/vendor/${name.replace(/.+\//, '')}/${type}`));
+    join(RootPath, `media/vendor/${name.replace(/.+\//, '')}/${type}`), { preserveTimestamps: true } );
```

Comment:
```suggestion
    join(RootPath, `media/vendor/${name.replace(/.+\//, '')}/${type}`), { preserveTimestamps: true });
```
CS fix for making drone happy.

Reply:
Thanks